packages/component/src/Utils/TypeFocusSink/getTabIndex.ts

```diff
@@ -0,0 +1,26 @@
+const { navigator: { userAgent } } = window;
+
+const IE_FOCUSABLE_LIST = ['a', 'body', 'button', 'frame', 'iframe', 'img', 'input', 'isindex', 'object', 'select', 'textarea'];
+
+const IS_FIREFOX = /Firefox\//i.test(userAgent);
```

Comment:
common location?

Reply:
This is the only browser check in whole Web Chat. I will keep them here for now. Try not to promote browser check as a standard in Web Chat.